packages/extension/src/ui/features/lock/LockScreen.tsx

```diff
@@ -63,9 +64,7 @@ export const LockScreen: FC = () => {
                 navigate(target)
                 return true
               } catch {
-                useAppState.setState({
-                  error: "Incorrect password",
-                })
+                setError("Invalid password")
```

Comment:
Should keep the same wording - 'invalid' implies the password contains unexpected characters
```suggestion
                setError("Incorrect password")
```